tests/pipelines/test_api.py

```diff
@@ -152,12 +155,12 @@ def _test_library_uploads(pipelines_api, api_method, spec, put_file_mock, dbfs_p
     ]
     expected_data['allow_duplicate_names'] = allow_duplicate_names
 
-    api_method(spec, allow_duplicate_names)
+    api_method(spec, tmpdir.strpath, allow_duplicate_names)
```

Comment:
it's not obvious, but `api_method` is either `create` or `deploy`. So this is the test that would actually fail.

Reply:
Ah, didn't realize that you switched the spec to use a relative path to itself rather than to the cwd.